### PR TITLE
Use `toml` for configuration files instead of `json`

### PR DESCRIPTION
**Pull request by madsmtm**
As discussed in https://github.com/areweguiyet/areweguiyet/issues/84.

I've removed the `frameworks` script, since I suspect that basically every single person that would be interested in contributing to this project already knows how to edit a toml file.

Additionally, I've begun a distinction between "crate name"/identifier and "what we display on the website". The former is important for links and such, where the latter is nice for having proper capitalization, or spaces in the name if relevant. There is still more work needed in this area, so treat that part as "work in progress".

### CODE REVIEW

**Comment by madsmtm** [comment]
I'll wait a few days for your input on this one @dxenonb, in case you have something to say

File: README.md [hunk 1]
```diff
@@ -10,73 +11,44 @@ replacing it with an automated system more like [lib.rs](https://lib.rs).
...
+### Submitting a crate
 
-**n.b.**, Because the CLI uses reqwest, which brings in hyper and tokio, it may take awhile to
-compile and requires almost a gigabyte of disk space.
+Add the crate to the `ecosystem.toml` file (using the keys documented
+therein), and open a pull request.
 
-### To submit a crate
+You are strongly encouraged to default to the crate's own defaults, which are
+written and updated by the crate maintainers themselves.
```
Comment (dxenonb) [inline README.md:32]: > You are strongly encouraged to default to the crate's own defaults, which are
written and updated by the crate maintainers themselves.

This should be rephrased slightly, something like:

> For crates on crates.io, you should only include `name` and `tags`, and leave all other fields blank. The site will populate the remaining fields with the crate's own defaults, which are written and updated by the crate maintainers themselves. AWGY is rebuilt monthly to keep these fields up to date (e.g. `docs`, `repo`)
>
> Your Pull Request will fail to build if `ecosystem.toml` includes these fields with the same data as crates.io.
>
> For projects/libraries/etc not on crates.io, or that are not well represented by a single crate, you can provide all the fields, along with `skip-crates-io` to prevent checking against crates.io.

File: README.md [hunk 2]
```diff
@@ -10,73 +11,44 @@ replacing it with an automated system more like [lib.rs](https://lib.rs).
...
 
-If you use the defaults, the CLI will automatically pull the most recent info from crates.io
-whenever `publish` is run.
 
-Now you're ready to open your PR!
+### Using the CLI
 
-### To submit a news link or post
+AreWeGuiYet uses a custom Rust CLI to fetch data from crates.io. It's
+currently a work in progress and is a little rough around the edges.
```
Comment (dxenonb) [inline README.md:43]: Since we don't intend for folks to touch the CLI anymore, we should maybe rephrase:

> AreWeGuiYet uses Github Actions to build the site. The Github Action is powered by a Rust CLI, which fetches the latest data for crates from crates.io.
>
> If you wish to build the site locally, building the CLI requires...

**Comment by dxenonb** [comment]
This looks great!

See my two comments above (and only the first one is important)

Thanks for getting this taken care of - feel free to merge

**Comment by madsmtm** [comment]
I think I've resolved your concerns, feel free to say so if they weren't. Thanks for the input!
